supplementary_style_guide/introduction/whats-new.adoc

```diff
@@ -32,6 +32,23 @@ To view the history of changes from 2020 through 2022, see the link:https://gith
 [[year-2024]]
 == 2024
 
+[[july-2024]]
+=== July 2024
+
+.Glossary entries
+* *xref:composite-content-view[composite content view]*: Updated glossary entry to lowercase spelling
+* *xref:content-view[content view]*: Updated glossary entry to lowercase spelling
+* *xref:nmstate[Nmstate]*: Added glossary entry
+* *xref:on-premise[on-premise]*: Updated glossary entry to include Red Hat OpenShift Container Platform definition
+* *Product*: Removed glossary entry
```

Comment:
Can we add a "why" for these removed entries, so people know why they were removed? For example, I wouldn't know that "Product" would have been related to Satellite.

Maybe:

```suggestion
* *Product*: Removed glossary entry because Red{nbsp}Hat Satellite no longer capitalizes it
```